backend/src/v4/middlewares/checkPermissions.js

```diff
@@ -124,22 +124,29 @@ function checkMultiplePermissions(permsRequest) {
 			models.push(...req.query.models.split(","));
 		}
 
-		validateUserSession(req, res).then(() => {
+		// PATCH Request
+		if(!models.length && req.body.length) {
+			models.push(...req.body.flatMap((entry)=> entry?.model || []));
+		}
+		try {
+			await validateUserSession(req, res);
+
 			const username = req.session.user.username;
 			const account = req.params.account;
 
-			const promises = [];
+			if(!models.length) {
```

Comment:
could this check be before the try ? or is there a reason its here ?

Reply:
oh I guess its to catch the error and return next(err)....

Reply:
yes 😆